src/commands/aksInspektorGadget/aksInspektorGadget.ts

```diff
@@ -0,0 +1,168 @@
+): Promise<void> {
+    const clustername = clusterInfo.name;
+    const kubeconfig = clusterInfo.kubeconfigYaml;
+
+    switch (gadget) {
+        case GadgetCommand.Deploy:
+            await deployGadget(clustername, kubeconfig, kubectl);
+            return;
+        case GadgetCommand.Undeploy:
+            const answer = await vscode.window.showInformationMessage(`Do you want to undeploy gadget in ${clustername}?`, "Yes", "No");
+            if (answer === "Yes") {
+                await unDeployGadget(clustername, kubeconfig, kubectl);
+            }
+            return;
+    }
+}
+
+async function deployGadget(
+    clustername: string,
+    clusterConfig: string,
+    kubectl: k8s.APIAvailable<k8s.KubectlV1>) {
+    const command = "deploy";
+
+    return await runKubectlGadgetCommands(clustername, command, clusterConfig, kubectl);
+}
+
+async function unDeployGadget(
+    clustername: string,
+    clusterConfig: string,
+    kubectl: k8s.APIAvailable<k8s.KubectlV1>) {
+    const command = "undeploy";
+
+    return await runKubectlGadgetCommands(clustername, command, clusterConfig, kubectl);
+}
+
+async function runKubectlGadgetCommands(
+    clustername: string,
+    command: string,
+    clusterConfig: string,
+    kubectl: k8s.APIAvailable<k8s.KubectlV1>) {
+
+    const kubectlGadgetPath = await getKubectlGadgetBinaryPath();
+
+    if (failed(kubectlGadgetPath)) {
+        vscode.window.showWarningMessage(`Gadget path is not found ${kubectlGadgetPath.error}`);
+        return;
+    }
+
+    const extensionPath = getExtensionPath();
+
+    if (failed(extensionPath)) {
+        vscode.window.showErrorMessage(extensionPath.error);
+        return;
+    }
+
+    return await longRunning(`Loading ${clustername} kubectl command run.`,
+        async () => {
+            const commandToRun = `gadget ${command}`;
+            const binaryPathDir = path.dirname(kubectlGadgetPath.result);
+
+            if (process.env.PATH === undefined) {
+                process.env.PATH = binaryPathDir
+            } else if (process.env.PATH.indexOf(binaryPathDir) < 0) {
+                process.env.PATH = binaryPathDir + path.delimiter + process.env.PATH;
+            }
+
+            const kubectlresult = await tmpfile.withOptionalTempFile<Errorable<k8s.KubectlV1.ShellResult>>(
+                clusterConfig, "YAML", async (kubeConfigFile) => {
+                    return await invokeKubectlCommand(kubectl, kubeConfigFile, commandToRun);
+                });
+
+            if (failed(kubectlresult)) {
+                vscode.window.showWarningMessage(`Gadget command failed with following error: ${kubectlresult.error}`);
+                return;
+            }
+
+            if (kubectlresult.succeeded) {
+                const webview = createWebView('AKS Kubectl Commands', `AKS Kubectl Command view for: ${clustername}`).webview;
+                webview.html = getWebviewContent(kubectlresult.result, command, extensionPath.result, webview);
```

Comment:
It should be `commandToRun`. In fact, it is showing `kubectl deploy` as command, instead of ``kubectl gadget deploy`:

```suggestion
                webview.html = getWebviewContent(kubectlresult.result, commandToRun, extensionPath.result, webview);
```